src/main/java/com/google/sps/servlets/quizTiming.java

```diff
@@ -0,0 +1,32 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List; 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+@WebServlet("/quizes")
+public class quizTiming extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    Entity quiz = new Entity("Quiz");
+    quiz.setProperty("timestamp", System.currentTimeMillis());
+    
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Saving a single datastore object in the constructor of this class would speed things up rather than making a new instance every time this path is called.